package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "talis-reading-lists-bookmarker",
-  "version": "0.3.6",
+  "version": "0.3.7",
   "description": "A browser extension to bookmark resources to Talis Aspire Reading Lists",
   "devDependencies": {
     "grunt": "^1.0.1",
-    "grunt-contrib-copy": "^1.0.0"
+    "grunt-contrib-copy": "^1.0.0",
+    "manifoldjs": "^0.7.6"
```

Comment:
how come we needed to add this? is it a new dependency? Or one that was there, but didn't exist in the package.json file?

Reply:
it's required (on the wiki page for packaging the ms extension) but we seem to be expecting ppl to do a `-g install` which is why I've added it as a devDependency and will update the wiki to reflect that